src/containers/guest-home-page/email-verification-modal/EmailVerificationModal.tsx

```
@@ -0,0 +1,37 @@
+  email: string
+}
+
+const EmailVerificationModal: React.FC<EmailVerificationModalProps> = ({
+  email
+}) => {
+  const { t } = useTranslation()
+
+  return (
+    <ImgTitleDescription
+      title={t('signup.confirmEmailTitle')}
+      description={
```

ESLint

[error] Props should be sorted alphabetically

(react/jsx-sort-props)